programmers-coding-test/wonji/greedy/숫자카드게임.js

```diff
@@ -0,0 +1,18 @@
+let N = prompt(`행의 개수 입력`);
+let M = prompt(`열의 개수 입력`);
+let data = prompt(`숫자들 입력`);
```

Comment:
브라우저 입력 방식이 아닌 커널 입력에 적응하는 것이 좋겠다.
```suggestion
const rl = require("readline-sync");

const MIN = 1;
const MAX = 10001;
const [ N, M ] = rl.question("입력해주세요.\n>> ").split(" ").map(val => parseInt(val));

const cards = [];
for (let i = 0; i < N; i++) {
  const card = []
  for (let j = 0; j < M; j++) {
    card.push(parseInt(Math.random() * (MAX - MIN) + MIN));
  } 
  cards.push(card);
}
```

이중 배열을 만들고자 할 때는 아래 처럼 입력받은 값을 배열안에 넣어주면된다.
```javascript
const datas = [];
for (let i = 0; i < N; i++) {
   for (let j = 0; j < M; j++) {
      const data = prompt(“입력”).split(“”); // split을 통해 배열로 만들어주었음.
   }
   datas.push( data ); // datas 배열에 data를 넣어주면 이차원 배열이 된다.
}
```